docs/assets.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 5
+sidebar_label: Asset Exchange
+title: Asset Exchange
+description: Exchange your treasury's tokens.
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Overview
+
+You can exchange your treasury's tokens to other supported assets and tokens, based on available swap providers.
+
+:::tip
+Near Treasury uses [RHEA Finance](concepts/exchange.md) for all asset exchange features. Learn more about it in [this article](concepts/exchange.md).
```

Comment:
May be we can resentence it, something like
"Near Treasury utilizes the Rhea Finance SDK, which enables token swaps through its Smart Routing feature."